.coveragerc

```diff
@@ -1,6 +1,5 @@
 [run]
 omit = .tox/*
-parallel = true
```

Comment:
Is the parallel option no longer supported? I feel like parallel would provide a good speedup for running tests,

Reply:
`parallel = true` interestingly doesn't actually control whether coverage runs in parallel. It makes it write to separate files so that coverage could be generated in parallel by an outer process.

What's odd is that `parallel = true` doesn't actually seem to do anything as far as I can tell. I suspect that `pytest-cov` is in charge and its `--cov-append` option is how they allow for parallelism.

Good point about wanting tox to run in parallel. I've updated this MR to now use `tox run-parallel --parallel-live`.